_build/03/1/Expressions.md

```diff
@@ -1,5 +1,6 @@
 ---
 interact_link: content/03/1/Expressions.ipynb
+download_link: /jupyter-book/assets/downloads/Expressions.ipynb.zip
```

Comment:
here these should be `Expressions.zip` (so strip the ipynb) since we'll also include things like images etc